tiny/config.yml

```diff
@@ -125,3 +125,6 @@ colors:
         fg: red
         bg: default
         attrs: [bold]
+
+# Show timestampe in the most left on every massage
+tsmsg: false
```

Comment:
Make this `always_show_timestamps` (or suggest another naming). It's OK (and even good) to be verbose here, as the file will be edited very rarely so users will have to remember this stuff every time they edit the file (happens to me when editing my vimrc and other configs).

As for documentation, I'd say "Always show timestamps of new messages".